src/android/toga_android/widgets/base.py

```diff
@@ -64,4 +84,17 @@ def add_child(self, child):
             child.container = self.container
 
     def rehint(self):
+        # An Android `View` subclass (of which all widgets are subclasses) can
+        # be measured against a measurement specification called a MeasureSpec.
+        # Note that this mutates the widget's `getMeasuredWidth()` and
+        # `getMeasuredHeight()`.
+        self.native.measure(
+            View__MeasureSpec.UNSPECIFIED, View__MeasureSpec.UNSPECIFIED
+        )
+        # We use `at_least()` below because without it, Travertino seems to
+        # decide to use a width of zero.
```

Comment:
Is this a case of *always* returns 0? Or only on initial iterations before physical display causes the widget to have size?

Reply:
What I know right now is that in manual testing, if I remove `at_least()` from the below line, the widget gets displayed with zero size. This is despite `self.native.getMeasuredWidth()` definitely returning a non-zero value for the width.

I'm not sure I'm being responsive to your question, but that's what I know, and I'm not sure I understand the question.

Reply:
Huh. That's interesting. Is that for all widgets, or just one of them?

Reply:
It seems to be true for all widgets. By this, what I mean is that when I remove `at_least()` from the call below (now duplicated across each widget), my Android app shows a blank screen. I believe (but haven't validated) that this is because `set_bounds()` is then being called with a `width` of `0`.